recipes/openmmtorchplugin/meta.yaml

```diff
@@ -0,0 +1,73 @@
+{% set name = "openmmtorchplugin" %}
+{% set version = "1.0.0" %}
+
+# see github.com/conda-forge/conda-forge.github.io/issues/1059 for naming discussion
+{% set torch_proc_type = "cuda" if cuda_compiler_version != "None" else "cpu" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  url: https://github.com/SCMusson/{{ name }}/archive/v{{ version }}.tar.gz
+  sha256: 1a9197c32209eaa37b9dcb1a7ef6ed30841ea5beed124b902485c20f5dd92021
+
+build:
+  number: 0
+  string: cuda{{ cuda_compiler_version | replace('.', '') }}py{{ CONDA_PY }}h{{ PKG_HASH }}_{{ PKG_BUILDNUM }}   # [cuda_compiler_version != "None"]
+  string: cpu_py{{ CONDA_PY }}h{{ PKG_HASH }}_{{ PKG_BUILDNUM }}                                                 # [cuda_compiler_version == "None"]
+  skip: true  # [cuda_compiler_version in (undefined, 'None', '10.2')]
+  skip: true  # [not unix]
+  # Set rpaths we need to link against libtorch.so, but it is located in site-packages
+  rpaths:
+    - lib/
+    - {{ SP_DIR }}/torch/lib
+  run_exports:
+    - {{ pin_subpackage( name|lower ) }}
+
+requirements:
+  build:
+    - {{ compiler('c') }}
+    - {{ compiler('cxx') }}
+    - {{ compiler('cuda') }}  # [cuda_compiler_version not in (undefined, 'None')]
+    - {{ cdt('mesa-libgl-devel') }}  # [linux]
+    - cmake
+    - make
+    - swig
+  host:
+    - khronos-opencl-icd-loader  # [osx]
+    - ocl-icd  # [linux]
```

Comment:
The [link inspector reports ](https://dev.azure.com/conda-forge/feedstock-builds/_build/results?buildId=517897&view=logs&j=1c012e3e-158a-5e5c-4fba-bceacec9f3ac&t=a2ee1cdc-f13c-52b8-e778-ce5a3d165825&l=2222)that ocl-icd is unused by your package. Is this copypasta?

Reply:
I removed it and it still works so yes

Reply:
You forgot the ones in the run section. If you aren't linking against ocl, then your package probably doesn't need any ocl components are runtime either.

Reply:
Ah sorry, I've removed the rest